<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Robot Arm Controller</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>20</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="Bluetooth">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>331</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Bluetooth Set</string>
    </property>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Equipment</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>73</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { 
    background-color: #4CAF50; 
    color: white; 
    border: none; 
    padding: 8px 16px; 
    border-radius: 4px; 
}
QPushButton:hover { 
    background-color: #45a049; 
}</string>
      </property>
      <property name="text">
       <string>Scan</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Status: Not connected</string>
      </property>
     </widget>
     <widget class="QPushButton" name="connect">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { 
    background-color: #2196F3; 
    color: white; 
    border: none; 
    padding: 8px 16px; 
    border-radius: 4px; 
}
QPushButton:disabled { 
    background-color: #cccccc; 
}</string>
      </property>
      <property name="text">
       <string>connect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="disconnect">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>0</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { 
    background-color: #f44336; 
    color: white; 
    border: none; 
    padding: 8px 16px; 
    border-radius: 4px; 
}
QPushButton:disabled { 
    background-color: #cccccc; 
}</string>
      </property>
      <property name="text">
       <string>disconnect</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="connection">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Control Mode</string>
    </property>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>231</width>
       <height>131</height>
      </rect>
     </property>
     <widget class="QPushButton" name="slider">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { 
    background-color: #FF9800; 
    color: white; 
    border: none; 
    padding: 16px; 
    border-radius: 8px; 
}
QPushButton:disabled { 
    background-color: #cccccc; 
}</string>
      </property>
      <property name="text">
       <string>Slider Control</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { 
    background-color: #9C27B0; 
    color: white; 
    border: none; 
    padding: 16px; 
    border-radius: 8px; 
}
QPushButton:disabled { 
    background-color: #cccccc; 
}</string>
      </property>
      <property name="text">
       <string>Direction Control</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Log">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>461</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection Log</string>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>411</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
